core/tasks.py

```
@@ -143,3 +160,55 @@ def generate_blog_post_suggestions(project_id: int):
     project.generate_title_suggestions(content_type=ContentType.SHARING, num_titles=3)
     project.generate_title_suggestions(content_type=ContentType.SEO, num_titles=3)
     return "Blog post suggestions generated"
+
+
+def try_create_posthog_alias(
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)